docs/research/multi-platform-scv.md

```
@@ -0,0 +1,574 @@
+```
+GET /api/v4/projects/:id/releases/:tag_name
+GET /api/v4/projects/:id/releases/:tag_name/assets/links
+```
+
+Release assets can be:
+- **Links** — external URLs attached to a release
+- **Sources** — auto-generated source archives
+- **Package Registry files** — linked from GitLab's built-in package registry
+
+Direct download URLs follow the pattern:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)